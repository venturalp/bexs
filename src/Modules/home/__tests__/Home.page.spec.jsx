import { render } from '@testing-library/react'
import { ThemeWrapper } from 'Commons/tests/Tests.Helpers'
import React from 'react'
import { HomePage } from '../Home.page'

describe('test module home', () => {
  it('render home', () => {
    const { asFragment } = render(
      <ThemeWrapper>
        <HomePage />
      </ThemeWrapper>,
    )

    expect(asFragment()).toMatchSnapshot()
  })
})
