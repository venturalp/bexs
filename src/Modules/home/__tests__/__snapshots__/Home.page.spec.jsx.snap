// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test module home render home 1`] = `
<DocumentFragment>
  <div
    class="sc-ieebsP fZACKJ"
  >
    <div
      class="sc-bkkfTU fqznwK"
    >
      <div>
        <store-logo.svg />
      </div>
      <nav
        class="sc-dJjZJu goZwhP"
      >
        <a
          href="#"
        />
        <a
          href="#"
        />
        <a
          href="#"
        />
        <a
          href="#"
        />
        <a
          href="#"
        />
      </nav>
    </div>
  </div>
  <div
    class="sc-bkkfTU fqznwK sc-hGPAah iAKuiz"
  >
    <div
      class="sc-ksdxAp fQhLgg"
    >
      <nav
        class="sc-crHlIS iEajBI"
      >
        <a
          class="sc-egiSv bbPwJg"
        >
          <arrow.svg
            width="15"
          />
          <p>
            Alterar forma de pagamento
          </p>
          <p>
            <strong>
              Etapa 2 
            </strong>
            de 3
          </p>
        </a>
        <div
          class="sc-bqiQRQ hnufKS"
        >
          <div>
            <card-icon.svg />
          </div>
          <h1>
            Adicione um novo 
            <br />
            cartão de crédito
          </h1>
        </div>
        <div
          class="sc-jrQzUz cOpTHM"
        >
          <img
            alt="img-hidden"
            class="img-hidden"
            src="card-front.png"
          />
          <div
            class="sc-pVTma egYZTx card-front"
          >
            <img
              alt="card"
              class="card"
              src="card-front.png"
            />
            <img
              alt="card"
              class="card empty"
              src="card-front-empty.png"
            />
            <div
              class="sc-kDThTU dywPpr"
            >
              <img
                alt="card-logo"
                src="visa-logo.png"
              />
              <div
                class="sc-furvIG iJqnIr"
              >
                <p>
                  ****
                </p>
                <p>
                  ****
                </p>
                <p>
                  ****
                </p>
                <p>
                  ****
                </p>
              </div>
              <div
                class="card-row"
              >
                <p />
                <p>
                  __/___
                </p>
              </div>
            </div>
          </div>
          <div
            class="sc-pVTma exfbzi card-back"
          >
            <img
              alt="card"
              class="card"
              src="card-back.png"
            />
            <img
              alt="card"
              class="empty card"
              src="card-back-empty.png"
            />
            <p
              class="sc-iqsfdx gtxooL"
            >
              ***
            </p>
          </div>
        </div>
      </nav>
      <form
        class="sc-jRQAMF edUICD"
      >
        <header>
          <div
            class="sc-iCfLBT lXOVz"
          >
            <div
              class="bullet"
            >
              <ico-check.svg
                classname="check"
              />
            </div>
            <p>
              Carrinho
            </p>
            <arrow.svg
              class="arrow"
            />
          </div>
          <div
            class="sc-iCfLBT iGYtDr"
          >
            <div
              class="bullet"
            >
              <p>
                2
              </p>
            </div>
            <p>
              Pagamento
            </p>
            <arrow.svg
              class="arrow"
            />
          </div>
          <div
            class="sc-iCfLBT iGYtDr"
          >
            <div
              class="bullet"
            >
              <p>
                3
              </p>
            </div>
            <p>
              Confirmação
            </p>
          </div>
        </header>
        <div
          class="sc-eCImvq iFSVdj"
        >
          <div
            class="sc-bdvvaa iLYldS"
          >
            <div
              class="sc-gsDJrp kxnSMu"
            >
              Número do cartão
            </div>
            <input
              data-testid="cardNumber"
              name="cardNumber"
            />
          </div>
          <p
            class="sc-dkPtyc fKvQaK"
          />
        </div>
        <div
          class="sc-eCImvq iFSVdj"
        >
          <div
            class="sc-bdvvaa iLYldS"
          >
            <div
              class="sc-gsDJrp kxnSMu"
            >
              Nome (igual ao cartão)
            </div>
            <input
              data-testid="cardName"
              name="cardName"
            />
          </div>
          <p
            class="sc-dkPtyc fKvQaK"
          />
        </div>
        <div
          class="row"
        >
          <div
            class="sc-eCImvq iFSVdj"
          >
            <div
              class="sc-bdvvaa iLYldS"
            >
              <div
                class="sc-gsDJrp kxnSMu"
              >
                Validade
              </div>
              <input
                data-testid="expiration"
                name="expiration"
              />
            </div>
            <p
              class="sc-dkPtyc fKvQaK"
            />
          </div>
          <div
            class="sc-eCImvq iFSVdj"
          >
            <div
              class="sc-bdvvaa iLYldS"
            >
              <div
                class="sc-gsDJrp kxnSMu"
              >
                <div
                  class="sc-gKckTs eMICKB"
                >
                  <p>
                    CVV
                  </p>
                  <ico-info.svg />
                </div>
              </div>
              <input
                data-testid="cvv"
                name="cvv"
              />
            </div>
            <p
              class="sc-dkPtyc fKvQaK"
            />
          </div>
        </div>
        <div
          class="sc-eCImvq iFSVdj"
        >
          <div
            class="sc-bdvvaa iLYldS"
          >
            <div
              class="sc-gsDJrp kxnSMu"
            >
              Parcelas
            </div>
            <select
              data-testid="instalments"
              name="instalments"
            >
              <option
                disabled=""
                selected=""
                value=""
              />
              <option
                value="1x"
              >
                1x R$ 12.000,00 sem juros
              </option>
              <option
                value="2x"
              >
                2x R$ 6.000,00 sem juros
              </option>
            </select>
          </div>
          <p
            class="sc-dkPtyc fKvQaK"
          />
        </div>
        <footer>
          <p
            class="sc-dkPtyc sc-hKwCoD fKvQaK eWfjcd"
          />
          <button
            type="submit"
          >
            Continuar
          </button>
        </footer>
      </form>
    </div>
    <div>
      <div
        class="sc-fotPbf dRUFRD"
      >
        <div
          class="title"
        >
          <div
            class="sc-hBURRC kiyPuZ"
            height="18"
          />
        </div>
        <hr
          class="sc-fFehDp kZLLgX"
        />
        <div
          class="row"
        >
          <div
            class="sc-hBURRC kezXoX"
          />
          <div
            class="sc-hBURRC kezXoX"
          />
        </div>
        <div
          class="row"
        >
          <div
            class="sc-hBURRC kezXoX"
          />
          <div
            class="sc-hBURRC kezXoX"
          />
        </div>
        <div
          class="row"
        >
          <div
            class="sc-hBURRC kezXoX"
          />
          <div
            class="sc-hBURRC kezXoX"
          />
        </div>
        <hr
          class="sc-fFehDp kZLLgX"
        />
        <div
          class="row"
        >
          <div
            class="sc-hBURRC eLLzdM"
          />
          <div
            class="sc-hBURRC eLLzdM"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
